docsite/source/built-in-types.html.md

```diff
@@ -15,7 +15,7 @@ Built-in types are grouped under 6 categories:
 
 ### Categories
 
-Assuming you included `Dry::Types` ([see instructions](docs::getting-started)) in a module called `Types`:
+Assuming you've defined your own `Types` module ([see instructions](docs::getting-started)):
```

Comment:
How do you feel about changing the assertion to one that would be more indicative that they _should_ do this, rather than conditional that they _might_ have.

```suggestion
After you've defined your own `Types` module ([see instructions](docs::getting-started)):
```